src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const firstNumber = Big(numberOne);
+  const secondNumber = Big(numberTwo);
+
+  switch (operation) {
+    case '+':
+      return firstNumber + secondNumber;
+    case '-':
+      return firstNumber - secondNumber;
+    case 'X':
```

Comment:
You might wanna use lowercase `x` to match the size with the rest of the operator.